### PR TITLE
Add test case for accessing customer view unauthorized

### PR DESCRIPTION
**Pull request by suutari-ai**
Refs SHOOP-1514

### CODE REVIEW

File: shoop_tests/front/test_customer_information.py [hunk 1]
```diff
@@ -52,3 +52,15 @@ def test_new_user_information_edit():
...
     assert get_user_model().objects.get(pk=user.pk).email == new_email
+
+
+@pytest.mark.django_db
+def test_user_information_unavailable_if_not_logged_in():
+    get_default_shop()  # Front middleware needs a Shop to exists
+    client = SmartClient()
+    customer_edit_url = reverse("shoop:customer_edit")
+    login_path = reverse("shoop:login")
+    login_path = (login_path[:-1] if login_path.endswith("/") else login_path)
```
Comment (akx) [inline shoop_tests/front/test_customer_information.py]: Huh, wait, why is this necessary?

File: shoop_tests/front/test_customer_information.py [hunk 2]
```diff
@@ -52,3 +52,15 @@ def test_new_user_information_edit():
...
     assert response.status_code == 302
     assert get_user_model().objects.get(pk=user.pk).email == new_email
+
+
+@pytest.mark.django_db
+def test_user_information_unavailable_if_not_logged_in():
+    get_default_shop()  # Front middleware needs a Shop to exists
+    client = SmartClient()
+    customer_edit_url = reverse("shoop:customer_edit")
+    login_path = reverse("shoop:login")
```
Comment (akx) [inline shoop_tests/front/test_customer_information.py]: This should probably be `resolve_url(settings.LOGIN_URL)`, actually ([that's what Django does](https://github.com/django/django/blob/1.8.4/django/contrib/auth/decorators.py#L24))

**Comment by suutari-ai** [comment]
@akx: Thanks for the comments. Your approach got the test much cleaner too. :)

Also removed some temporary variables for brevity.
